lib/topological_inventory.rb

```diff
@@ -14,7 +14,9 @@ def self.call
 
   private_class_method def self.raw_api
     TopologicalInventoryApiClient.configure do |config|
-      config.host   = ENV['TOPOLOGICAL_INVENTORY_URL'] || 'localhost'
+      config.host = ENV['TOPOLOGICAL_INVENTORY_URL'] || 'localhost'
+      config.username = ENV['TOPOLOGICAL_INVENTORY_USERNAME'] || nil
```

Comment:
@lindgrenj6 @syncrou 
This is only needed when you are doing local development, in QE and production we are getting authenticated via Red Hat Insights & 3 Scale gateway. Once we are authenticated intra service communication is done by passing the   proper auth headers.

Reply:
@syncrou I spoke with @mkanoor as well, I added some documentation to the README as well as changing it to only set these if running in the dev environment.